expression/builtin_compare.go

```diff
@@ -2540,21 +2540,21 @@ func (b *builtinNullEQIntSig) evalInt(row chunk.Row) (val int64, isNull bool, er
 	case isNull0 && isNull1:
 		res = 1
 	case isNull0 != isNull1:
-		break
+		return res, false, nil
 	case isUnsigned0 && isUnsigned1 && types.CompareUint64(uint64(arg0), uint64(arg1)) == 0:
 		res = 1
 	case !isUnsigned0 && !isUnsigned1 && types.CompareInt64(arg0, arg1) == 0:
 		res = 1
 	case isUnsigned0 && !isUnsigned1:
 		if arg1 < 0 {
-			break
+			return res, false, nil
 		}
 		if types.CompareInt64(arg0, arg1) == 0 {
 			res = 1
 		}
 	case !isUnsigned0 && isUnsigned1:
 		if arg0 < 0 {
-			break
+			return res, false, nil
 		}
 		if types.CompareInt64(arg0, arg1) == 0 {
 			res = 1
```

Comment:
```break``` in the switch is so weird. it has two way to fix it.

 1、diectly replace with ```return```
 2、remove this switch branch. some of ```switch``` branch is useless.
 3、refactor the code
 
I do minimal changes to keep code review easy. but I think it still needs to improve.

Reply:
BTW, Adding ```break``` in the ```switch``` is a code style of ```C/C++```？

Reply:
> BTW, Adding `break` in the `switch` is a code style of `C/C++`？

Of course. Since `case` in `c/cpp` will fall-through by default, so it is natural to use `break` everywhere. It is a like `return` for `switch` block.

Go does not need to use `break` every `case`, but you can still use `break` to "return from the switch block".

Reply:
I think this change is fine because the code is simple enough.